gemfiles/rails_4_2.gemfile

```diff
@@ -3,12 +3,11 @@
 source "https://rubygems.org"
 
 gem "rails", "~> 4.2.0"
-gem "bcrypt", "~> 3.1", require: false
 gem "appraisal"
+gem "bcrypt", "~> 3.1", require: false
 gem "activerecord-jdbcsqlite3-adapter", platform: :jruby
 gem "sqlite3", "~> 1.3", "< 1.4", platform: [:ruby, :mswin, :mingw, :x64_mingw]
 gem "tzinfo-data", platforms: [:mingw, :mswin, :x64_mingw]
-# Older Grape requires Ruby >= 2.2.2
```

Comment:
Don't see any reasons to drop this comment till we drop support of Ruby 2.2

Reply:
Remove this because the file `# This file was generated by Appraisal`